Makefile

```diff
@@ -1,7 +1,7 @@
 SHELL=/usr/bin/env bash
 
 EMACS ?= emacs
-EASK ?= eask
+EASK ?= eask --verbose 4
 
 build:
 	$(EASK) package
```

Comment:
Try add `$(EASK) install-deps` before `$(EASK) package`.

Reply:
Lol, genius. Why has this been working before?

Reply:
I'm not exactly sure why, but there is certainly a bug in Emacs 27.x. It could be a conflict  (jsonrpc) with their built-in packages?

Anyways, `eask install-deps` will assumed the user will always use the latest packages/dependencies. So that resolved the conflict from version base packages? (again, I could be wrong)

Reply:
But in that case 28 should also have hit the error? Weird.

Thanks for the spot in any case!